lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -104,6 +104,11 @@
             - Wait until vCenter detects an IP address for the VM
             - This requires vmware-tools (vmtoolsd) to properly work after creation
         default: False
+   snapshot_name:
```

Comment:
snapshot_src is better name i think for linked clone, it's unclear for non linked clones usage